modules/bridge/BUILD

```diff
@@ -0,0 +1,70 @@
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "bridge_control_lib",
+    srcs = [
+    ],
+    copts = ['-DMODULE_NAME=\\"bridge\\"'],
+    deps = [
+        "//cyber",
+        "//external:gflags",
+        "//modules/canbus/proto:canbus_proto",
+        "//modules/common",
+        "//modules/common/adapters:adapter_gflags",
+        "//modules/common/monitor_log",
+        "//modules/common/time",
+        "//modules/common/util",
+        "//modules/control/common",
```

Comment:
why this depends on sub-folder under control? It should only be depending on `control_proto` right?